sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Create-disbursement-overawards-table.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS sims.disbursement_overawards(
+  id SERIAL PRIMARY KEY,
+  student_id INT NOT NULL REFERENCES sims.students(id),
+  student_assessment_id INT REFERENCES sims.student_assessments(id),
+  disbursement_schedule_id INT REFERENCES sims.disbursement_schedules(id),
+  overaward_value NUMERIC(8, 2) NOT NULL,
+  disbursement_value_code VARCHAR(10) NOT NULL,
+  origin_type sims.disbursement_overaward_origin_types NOT NULL,
+  deleted_at TIMESTAMP WITH TIME ZONE,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL,
+    modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.disbursement_overawards IS 'Students overawards resulted from reassessments calculations.';
+
+COMMENT ON COLUMN sims.disbursement_overawards.id IS 'Auto-generated sequential primary key column.';
+
+COMMENT ON COLUMN sims.disbursement_overawards.student_id IS 'Student related to the overaward.';
+
+COMMENT ON COLUMN sims.disbursement_overawards.student_assessment_id IS 'Related the student assessment. When not present, it represents a manual entry.';
+
+COMMENT ON COLUMN sims.disbursement_overawards.disbursement_schedule_id IS 'Related disbursement schedule. When not present, it represents a manual entry or an overaward at the assessment level, not at the disbursement schedule level (award).';
```

Comment:
`an overaward at the assessment level,` means

Reply:
It means that it is not related to an award. 
When an overaward is deducted from an award it will have an impact on the overaward balance from an "award perspective".
When an overaward is generated it will have an impact on the overaward balance from an assessment perspective.